packages/destination-actions/src/destinations/google-analytics-4/customEvent/index.ts

```diff
@@ -2,7 +2,14 @@ import type { ActionDefinition } from '@segment/actions-core'
 import type { Settings } from '../generated-types'
 import type { Payload } from './generated-types'
 import { client_id } from '../ga4-properties'
+const normalize_event_name = (name: string, lowercase: boolean): string => {
+  name = name.replace(/\s/g, '_')
 
+  if (lowercase === true) {
```

Comment:
I think simply checking whether `lowercase` is truthy should be ok here.

```suggestion
  if (lowercase) {
```